### DISCUSSION TITLE
ArrayTabs 每个Tabs能否支持不同schema

### DISCUSSION THREAD

**Discussion by Nokecy** [alibaba/formily#1661] [Q&A 2.x] [answered]
https://github.com/alibaba/formily/blob/f73deaf6724e02673590648e98f19e6e917ccc46/packages/antd/src/array-tabs/index.tsx#L61
 我看到这里有判断items是否为数组，使用Markup Schema该怎么组织代码~~~ 求大佬指教

**Comment by janryWang** [comment, marked as answer]
类型是支持的，组件不支持，暂时不想支持这么复杂的场景

**Reply by janryWang** [reply]
还有一个我不想支持的原因，就是这种描述对设计器不友好

**Reply by Nokecy** [reply]
ok ~~~ 如果我想自定义的话 ，有没有方向指点一下。~~
